apps/wallet-mobile/src/features/Settings/ApplicationSettings/ApplicationSettingsScreen.tsx

```diff
@@ -0,0 +1,223 @@
+      <ScrollView bounces={false}>
+        <StatusBar type="light" />
+
+        <SettingsSection title={strings.general}>
+          <NavigatedSettingsItem
+            icon={<Icon.Globe {...iconProps} />}
+            label={strings.selectLanguage}
+            navigateTo="change-language"
+            selected={language.label}
+          />
+
+          <NavigatedSettingsItem
+            icon={<Icon.Coins {...iconProps} />}
+            label={strings.selectFiatCurrency}
+            selected={currency}
+            navigateTo="change-currency"
+          />
+
+          <NavigatedSettingsItem icon={<Icon.Info {...iconProps} />} label={strings.about} navigateTo="about" />
+
+          <NavigatedSettingsItem
+            icon={<Icon.TermsOfUse {...iconProps} />}
+            label={strings.termsOfservice}
+            navigateTo="terms-of-use"
+          />
+        </SettingsSection>
+
+        <Spacer height={24} />
+
+        <SettingsSection title={strings.securityReporting}>
+          <NavigatedSettingsItem
+            icon={<Icon.Pin {...iconProps} />}
+            label={strings.changePin}
+            navigateTo="change-custom-pin"
+          />
+
+          <SettingsItem
+            icon={<Icon.EyeOff {...iconProps} />}
+            label={strings.privacyMode}
+            info={strings.privacyModeInfo}
+          >
+            <PrivacyModeSwitch />
+          </SettingsItem>
+
+          <SettingsItem
+            icon={<Icon.Bio {...iconProps} />}
+            label={strings.biometricsSignIn}
+            info={strings.biometricsSignInInfo}
+            disabled={!authOsEnabled}
+          >
+            <Switch
+              value={authSetting === 'os'}
+              onValueChange={onToggleAuthWithOs}
+              disabled={!authOsEnabled || isTogglePrivacyModeLoading}
+            />
+          </SettingsItem>
+
+          <SettingsItem
+            icon={<Icon.Export {...iconProps} />}
+            label={strings.crashReporting}
+            info={strings.crashReportingInfo}
+          >
+            <Switch
+              value={crashReports.enabled}
+              onValueChange={crashReports.enabled ? crashReports.disable : crashReports.enable}
+              disabled={isNightly() || isTogglePrivacyModeLoading}
+            />
+          </SettingsItem>
+        </SettingsSection>
+      </ScrollView>
+    </SafeAreaView>
+  )
+}
+
+const PrivacyModeSwitch = () => {
+  const {isPrivacyOff, togglePrivacyMode, isTogglePrivacyModeLoading} = usePrivacyMode()
+  const [isLocalPrivacyOff, setIsLocalPrivacyOff] = React.useState(isPrivacyOff)
+
+  const onTogglePrivacyMode = (switchValue) => {
+    setIsLocalPrivacyOff(switchValue)
+    togglePrivacyMode()
+  }
+
+  React.useEffect(() => {
+    setIsLocalPrivacyOff(isPrivacyOff)
+  }, [isPrivacyOff])
+
+  return <Switch value={isLocalPrivacyOff} onValueChange={onTogglePrivacyMode} disabled={isTogglePrivacyModeLoading} />
+}
```

Comment:
To avoid switch "jumps"


https://github.com/Emurgo/yoroi/assets/105349292/a164d068-ba0f-423d-a492-9cf0321132ca

Reply:
I'd say the issue is the lack of the boundary while performing the transition, but this solution gives a smoother UI experience, I'd only replace Local to displayed to keep a convention, `isPrivacyOffDisplayed`

Reply:
refactored a bit. now i will add your suggestion

Reply:
https://github.com/Emurgo/yoroi/pull/2563/files#diff-0b27a317934ceb3a8f54f0edc0de254662a2dd3e7f134790aec9e9f68842582bR127